docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,97 @@
+# Adaptive concurrency limits
+
+BigCommerce aims to provide outstanding platform reliability. To maintain dependability, adaptive concurrency limits are applied to both REST and GraphQL APIs. These limits enhance quality for merchants by regulating multiple API calls. In this doc we'll outline how these limits work on the BigCommerce platform.
+
+<Callout type="info">
+  BigCommerce recommends adhering to industry-standard rate limiting when developing. 
+</Callout>
+
+## How limits work
+
+Each API request consumes one quota unit. When the quota is reached, subsequent requests are rejected until the quota is refreshed. 
+A store's quota is shared across all apps accesing the store, preventing a single app from using the entire quota. The available quota for an app adjusts as other clients make or stop requests. 
+
+Try locating an API header using CLI. Use the `curl` command to find the rate limit for the endpoint in this example. 
+
+``` bash filename="Example: cURL command" showLineNumbers copy
+    curl -I https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/catalog/products
+```
+
+The maximum quota limit is returned in the API response header.
+
+### API response headers
```

Comment:
I realize now, our endpoints don't have that information in headers. How will they locate their quota limits again?